bukkit/src/main/java/haveric/stackableItems/util/SIItems.java

```diff
@@ -309,10 +308,10 @@ public static int getItemMax(Player player, Material mat, short dur, InventoryTy
                 // Check player
                 String uuid = player.getUniqueId().toString();
 
-                max = getMax(world, "player", uuid, mat, dur);
+                max = getMax(world, "player", uuid, mat, durability);
```

Comment:
In a couple areas renamed "dur" to "durability". Didn't rename all of them yet, but should. Much more descriptive this way